feature-libs/product/multi-dimensional/core/model/augmented-core.model.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Image } from '@spartacus/core';
+
+export type VariantCategoryOption = {
+  value: string;
+  code: string;
+  images: Image[];
+  order: number;
+};
+
+export type VariantCategory = {
+  name: string;
+  variantOptions: VariantCategoryOption[];
+  hasImages: boolean;
+  order: number;
+};
+
+declare module '@spartacus/core' {
+  const enum ProductScope {}
+}
```

Comment:
[MEDIUM]
Models don't have weight of runtime JS. So they can be safely put in to the `/root` entry point. Moreover, it feels right to put them to `/root`, becasue potentially any other library can then simply import types from multidimensional `/root` entry pount without breaking lazy laoding of multidimensional feature.

Reply:
moved to `root`